search-api.md

```diff
@@ -88,6 +90,15 @@ After receiving a request, the remote server can respond in one of two ways:
 }
 ```
 
+#### Disclaimer
+* *Optional*
+* Disclaimer text for the service, this is optional and can be included in all requests and responses. If included it is assumed that it supersedes the disclaimer stored in the GitHub repository.
+* Note that the diclaimer should be shown with the returned data per item 6 of the MME service agreement.
```

Comment:
I didn't think this was required as item 6 doesn't say anything extra, and an implementor should have already read and agreed to the service agreement.